src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  padding-top: 2px;
+}
+
+.search__bar {
+  font-family: Avenir, Helvetica, sans-serif;
+  margin: 10px 0;
+  width: 100%;
+  box-sizing: border-box;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big {
+  font-size: 16px;
+  height: 70px;
+  line-height: 70px;
+  padding-left: 62px;
+}
+
+.small {
+  padding-top: 2px;
+  font-size: 14px;
+  height: 42px;
+  line-height: 40px;
+  padding-left: 33px;
+}
+
+.search__bar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  font-weight: 400;
+}
+
+.search__bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.form {
+  /* margin-left: 40px; */
+  position: relative;
+}
+
+.big__form::before {
+  height: 19px;
+  width: 19px;
+  content: "";
+  background-image: url("images/Search.svg");
+  background-size: contain;
+  position: absolute;
+  background-repeat: no-repeat;
+  top: 35px;
+  left: 26px;
+}
+
+.small__form::before {
+  height: 11px;
+  width: 11px;
+  content: "";
+  background-image: url("images/Search.svg");
+  background-size: contain;
+  position: absolute;
+  background-repeat: no-repeat;
```

Comment:
these properties are the same for both - you can move them to `.form`

Reply:
It's doesn't work. I don't know, how to make it work.